cli/internal/run/run.go

```diff
@@ -394,7 +394,8 @@ type runOpts struct {
 	// Whether to emit a perf profile
 	profile string
 	// Immediately exit on task failure
-	bail            bool
+	//bail            bool
```

Comment:
Change request: update the "// Immediately exit on task failure" comment and remove the commented-out `bail`.

Reply:
Done.